### ISSUE TITLE
Generate a report of differences between databases

### ISSUE THREAD

**Issue by MarkGaze** [OpenDBDiff/OpenDBDiff#81]
We can currently compare two databases and see the results in the GUI but I'd like to be able to share the differences as a report to other (non-technical) stakeholders.

I'm thinking some sort of HTML report that would emulate the GUI, with the ability to look at the diff in a separate page for each file.

Is this something that would be considered?

**Comment by igitur** [comment]
I agree. I think this is definitely necessary. I would suggest rather `.xlsx` first and maybe `.html` later. Which parts of the GUI do you think should be in there? Mirroring the treeview should be simple, but including all the script diffs will complicate it a bit.

**Comment by MarkGaze** [comment]
Yeah, `.xlsx` with a mirror of the treeview would be great as a starting point. As it would highlight the files that have changed and then I could refer back to the actual tables/stored procs to check the differences.

I'm effectively trying to highlight hotspots where there could be a problem between versions of the database so that would work for me 😄 

A bonus would be being able to output this on the command line too!

**Comment by igitur** [comment]
I'm thinking reports should be generated in any of these formats chosen by the user:
- csv
- xlsx
- xml
- html (generated by transformating xml)

**Comment by MarkGaze** [comment]
I agree with that list

**Comment by Mops33** [comment]
This would be the main missing feature for us.
Report format could be Excel or HTML, but csv and xml would be difficult to communicate across teams.
